django/contrib/admin/static/admin/js/inlines.js

```diff
@@ -296,7 +296,14 @@
                     dependency_list = input.data('dependency_list') || [],
                     dependencies = [];
                 $.each(dependency_list, function(i, field_name) {
-                    dependencies.push('#' + row.find('.form-row .field-' + field_name).find('input, select, textarea').attr('id'));
+                    let field_element = row.find('.form-row.field-' + field_name + ' .field-' + field_name).find('input, select, textarea');
+                   
+                    // If length is zero, then the field is not inside a fieldset
+                    if (!field_element.length) {  
+                        field_element = row.find('.form-row.field-' + field_name).find('input, select, textarea');
+                    }
+                    
+                    dependencies.push('#' + field_element.attr('id'));
```

Comment:
Can we move `.find('input, select, textarea')` to the `dependencies.push()`?:
```suggestion
                    let field_element = row.find('.form-row.field-' + field_name + ' .field-' + field_name);
                    // If length is zero, then the field is not inside a fieldset
                    if (!field_element.length) {  
                        field_element = row.find('.form-row.field-' + field_name);
                    }                   
                    dependencies.push('#' + field_element..find('input, select, textarea').attr('id'));
```

Reply:
Yeah, I think we can. I will do that.